testdata/cheats/Cheats.sol

```diff
@@ -135,4 +135,27 @@ interface Cheats {
     function toString(bool)           external returns(string memory);
     function toString(uint256)        external returns(string memory);
     function toString(int256)         external returns(string memory);
+    // Snapshot the current state of the evm.
+    // Returns the id of the snapshot that was created.
+    // To revert a snapshot use `revertTo`
+    function snapshot() external returns(uint256);
+    // Revert the state of the evm to a previous snapshot
+    // takes the snapshot id to revert to. This deletes the snapshot and all snapshots taken after the given snapshot id.
+    function revertTo(uint256) external returns(bool);
+    // Creates a new fork with the given endpoint and block and returns the identifier of the fork
+    function createFork(string calldata,uint256) external returns(uint256);
+    // Creates a new fork with the given endpoint and the latest block and returns the identifier of the fork
+    function createFork(string calldata) external returns(uint256);
+    // takes a fork identifier created by `createFork` and changes the state
+    function selectFork(uint256) external;
+    // Updates the currently active fork to given block number
+    // This is similar to `roll` but for the fork
+    function rollFork(uint256) external;
```

Comment:
Just a smol question, how does this work if you do *not* use any of the fork cheatcodes, but just use the CLI args? Still as if you had used one of the cheatcodes?

Reply:
exactly, initializing with CLI args will be the same as creating and selecting during setup